Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ruby:2.5.3-alpine
+
+# postgresql-client is required for invoke.sh
+RUN apk --no-cache add \
+  postgresql-dev \
+  postgresql-client \
+  tzdata \
+  libxml2-dev \
+  libxslt-dev
+
+LABEL maintainer="Aaron Collier <aaron.collier@stanford.edu>"
+
+RUN mkdir /app
+WORKDIR /app
+
+# Using argument for conditional setup in conf file  
+ARG RAILS_ENV
```

Comment:
I'm fuzzy on this, but RAILS_ENV is an ARG here but it's not listed as an arg in docker-compose.yml. Oversight or unnecessary?

Reply:
Unnecessary, I think. This allows to bake in RAILS_ENV later when we start pushing built images (using --build-arg). Carry on!

Reply:
Yeah, this allows it to be passed/set in docker-compose or the command line